website/docs/from_provider/quickstart.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Quickstart
+---
+
+## Try Riverpod online
+
+Before anything, we'd suggest you try out Riverpod in a sandbox to test its features out.
+
+The following example should be familiar and readable enough:
+  - dartpad.dev example (link)
+  - marvel app example, or another github repo example (link)
+  - ... more?
+
+## Migrating
+
+This section is designed for people familiar with the [Provider] package who
+wants to learn about Riverpod.
+
+Migrating from Provider to Riverpod can be very straightforward.  
+Migrating basically consists in a few steps that can be done in an *incremental* way.
+
+### Start gently
+*If you have an existing app, don't try to migrate all your providers at once.*  
+Don't immediately try to change your `ChangeNotifiers` into the more modern [AsyncNotifiers] from Riverpod.
+This will requite a bit of a paradigm shift, so it may be difficult to do initially.  
+
+Take your time, as it is important to get yourself familiar with Riverpod first.  
+The following is perfectly fine to start with:
+
+```dart
+// If you have this...
+class MyNotifier extends ChangeNotifier {
+  int state = 0;
+
+  void increment() {
+    state++;
+    notifyListeners();
+  }
+}
+
+// ... just add this!
+final myNotifierProvider = ChangeNotifierProvider<MyNotifier>((ref) {
+  return MyNotifier();
+});
+```
+
+It's fine to keep using `ChangeNotifier` using Riverpod and not use the latest fancy Riverpod features ASAP.
+
+While you should strive toward moving all your application to Riverpod in the long-run, 
+**don't burn yourself out**.  
+Do it one provider at a time.  
+
+### Riverpod and Provider can coexist
+*Keep in mind that it is entirely possible to use both Provider and Riverpod at the same time.*
+
+Indeed, using import aliases, it is possible to use the two APIs altogheter.  
+This is also great for readibilty and it removes any ambiguous API usage.
+
+If you plan on doing this, consider using import aliases for each Provider import in your codebase.
+
+:::info
+A full guide onto how to effectively implement import aliases is incoming soon.
+:::
+
+### Migrate one Provider at a time
+*All Providers from pkg:provider have a strict equivalent in Riverpod*.
```

Comment:
Technically no if you consider ProxyProvider variants

Reply:
We're missing that migration btw.

The migration involves using ref.listen

Reply:
> The migration involves using ref.listen

Could you elaborate on this? Is mentioning the fact that Providers in pkg:Riverpod are composable by default enough? Or do you have something else in mind? (:

Reply:
> We're missing that migration btw.

I've added a small paragraph in the quickstart, cross referencing the "deeper" article. Let me know if that's what you intended.